staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -475,7 +475,7 @@ func (t *tracker) Delete(gvr schema.GroupVersionResource, ns, name string) error
 // filterByNamespaceAndName returns all objects in the collection that
 // match provided namespace and name. Empty namespace matches
 // non-namespaced objects.
-func filterByNamespaceAndName(objs []runtime.Object, ns, name string) ([]runtime.Object, error) {
+func filterByNamespaceAndName(objs []runtime.Object, ns, name string, list bool) ([]runtime.Object, error) {
```

Comment:
Can you add a unit test?

Reply:
Can you add a unit test to avoid regression?

Reply:
`list bool` reads somewhat confusingly... could we use a more descriptive parameter name here?

Reply:
I moved the check outside of `filterByNamespaceAndName` and directly into `tracket.Get` method instead. So `filterByNamespaceAndName` stays unchanged.